cobigen-templates/crud-ionic-client-app/src/main/resources/templates.xml

```diff
@@ -1,72 +1,68 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<templatesConfiguration xmlns="http://capgemini.com/devonfw/cobigen/TemplatesConfiguration"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1">
-
+<templatesConfiguration xmlns="http://capgemini.com/devonfw/cobigen/TemplatesConfiguration" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.3">
```

Comment:
The latest version for the templates.xml will be 5.0, right? So the version does not match. Also for the other template configurations